deploy/charts/crate-operator/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: crate-operator
+description: Crate Operator - Helm chart for installing and upgrading Crate Operator.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "2.13.3"
```

Comment:
Let's also update the release procedure to change this when we release a new version of the operator?